tests/Http/Deserializer/Calendar/CalendarJSONParserTest.php

```diff
@@ -93,28 +98,55 @@ public function it_returns_null_when_end_date_is_missing()
     /**
      * @test
      */
-    public function it_can_get_the_time_spans()
+    public function it_can_get_the_status()
```

Comment:
```suggestion
    public function it_can_get_the_status(): void
```

Reply:
There are several other instances of this missing. I'm not sure if it's worth it to add them all now.

Reply:
Easy enough to modify for the touched files in this pull request.